src/css/components/sidenav.scss

```diff
@@ -150,76 +146,123 @@ $color-heading:     $color-primary-alt-light;
   transform: rotate(45deg);
 }
 
-.sidenav-active {
-  background-color: $color-highlight;
-  color: $color-background;
+.sidenav-level-one .sidenav-active {
 
   > a {
-    color: $color-background;
-    background-color: $color-highlight;
-    font-weight: 800;
     text-decoration: none;
+    cursor: default;
 
-    &:hover {
-      color: $color-background;
-      background-color: $color-highlight;
+    &:hover,
+    &:focus {
+      color: $color-blue;
     }
-  }
 
-  /* stylelint-disable declaration-no-important */
-  .sidenav-arrow {
-    border-color: $color-background !important; // prevents hover re-color when active
+    .sidenav-arrow {
+      border-color: $color-blue;
+    }
   }
-  /* stylelint-enable */
 }
 
 .sidenav-level-two {
-  background-color: $color-background;
+  a {
+    margin-left: 9px;
+    position: relative;
+
+    &:before {
+      background-color: $color-lightgray;
+      bottom: 0;
+      content: '';
+      display: block;
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: $grid-05;
+    }
+  }
 
-  > li {
-    background-color: $color-background;
-    margin-bottom: 0;
+  > li:first-child > a:before {
+    border-top-left-radius: $grid-1;
+    border-top-right-radius: $grid-1;
+    top: .1em;
+    margin-top: $grid-05;
+  }
+
+  > li:last-child > a:before {
+    border-bottom-left-radius: $grid-1;
+    border-bottom-right-radius: $grid-1;
+    bottom: .3em;
+  }
 
-    a {
-      padding-left: 2rem;
+  .sidenav-active {
+    > a {
+      color: $color-textblack;
 
       &:hover {
-        padding-left: 2rem;
+        color: $color-textblack;
+      }
+
+      &:before {
+        background-color: $color-textblack;
       }
     }
   }
+
+  .sidenav-active-dashboard {
```

Comment:
this can be deprecated (and returned to `sidenav-active`) as the functionality is now the same